pkg/volume/secret/secret_test.go

```diff
@@ -124,8 +123,7 @@ func TestPlugin(t *testing.T) {
 	doTestSecretDataInVolume(volumePath, secret, t)
 
 	metrics, err := builder.GetMetrics()
-	assert.NotEmpty(t, metrics)
-	assert.NoError(t, err)
+	volume.AssertMetricsOk(t, metrics, err)
```

Comment:
Instead of skipping the rest of the test, what do you think of just logging the error.